pkgs/development/python-modules/drf-extra-fields/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  django,
+  djangorestframework,
+  filetype,
+  pytestCheckHook,
+  pytest-django,
+}:
+
+buildPythonPackage rec {
+  pname = "drf-extra-fields";
+  version = "3.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "hipo";
+    repo = "drf-extra-fields";
+    rev = "v${version}";
+    hash = "sha256-Ym4vnZ/t0ZdSxU53BC0ducJl1YiTygRSWql/35PNbOU";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    django
+    djangorestframework
+    filetype
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-django
```

Comment:
The tests seem to require psycopg2 (see ofborg logs) but that's undocumented. Maybe in the meantime we got a django update? don't really know. https://github.com/Hipo/drf-extra-fields/blob/8c18a7542c8a38fe3dccd1874a74a38410aa3a7f/drf_extra_fields/compat.py#L6